scripts/launch-prover-service

```diff
@@ -4,20 +4,24 @@
 # client contract will be deployed. This feature is intended for demo or testnet
 # purposes only.
 set -eo pipefail
+NUM_BLOCKS_PER_EPOCH=10
```

Comment:
Is there a better place to set the value of this variable? If it's interesting to change this I think there should be an env var and if not, shouldn't we hard-code it in the `runDemo` function?

Reply:
Environment variable added in b182b5b.